content/docs/tasks/telemetry/access-log/index.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Envoy's Access Logging
+description: This task shows you how to configure Envoy proxies to print access log to their standard output.
+weight: 9
+keywords: [telemetry]
+---
+
+The simplest kind of Istio logging is
+[Envoy's access logging](https://www.envoyproxy.io/docs/envoy/latest/configuration/access_log).
+Envoy proxies print access information to their standard output.
+The standard output of Envoy's containers can then be printed by the `kubectl logs` command.
+
+{{< boilerplate before-you-begin-egress >}}
+
+{{< boilerplate start-httpbin-service >}}
+
+## Enable Envoy's access logging
+
+Edit the `istio` configuration map:
+
+{{< text bash >}}
+$ kubectl edit cm istio -n istio-system
+{{< /text >}}
+
+Set the value of `global.proxy.accessLogFile` to "/dev/stdout".
+Mind escaping quotation marks with backward slashes (`\"`).
+
+You may also want to change the
+[format](https://www.envoyproxy.io/docs/envoy/latest/configuration/access_log#format-rules) of the access log.
+To do it, edit `global.proxy.accessLogFormat`.
+
+Save the configuration map and exit the editing mode.
+
+## Test the access log
+
+1.  Send a request from `sleep` to `httpbin`:
+
+    {{< text bash >}}
+    $ kubectl exec -it $(kubectl get pod -l app=sleep -o jsonpath={.items..metadata.name}) -c sleep -- curl -v httpbin:8000/status/418
+    *   Trying 172.21.13.94...
+    * TCP_NODELAY set
+    * Connected to httpbin (172.21.13.94) port 8000 (#0)
+    > GET /status/418 HTTP/1.1
+
+    ...
+    < HTTP/1.1 418 Unknown
+    < server: envoy
+    ...
+
+        -=[ teapot ]=-
+
+           _...._
+         .'  _ _ `.
+        | ."` ^ `". _,
+        \_;`"---"`|//
+          |       ;/
+          \_     _/
+            `"""`
+    * Connection #0 to host httpbin left intact
+    {{< /text >}}
+
+1.  Check the log of `sleep`:
```

Comment:
Check `sleep`'s log:

Reply:
Fixed in https://github.com/istio/istio.io/pull/3531/commits/cfad13d46485801dbc2b3cd26592088e0b29e7f9